docs/Userguide_python.rst

```diff
@@ -105,3 +105,44 @@ can be good to periodically clean up Conda's cache:
 .. prompt:: bash (<env>)$
 
    conda clean --all
+
+**Mamba**
+
+When installing new packages with ``conda install``, conda uses a built-in 
+dependency solver for solving the dependency graph of all packages (and their 
+versions) requested such that package dependency conflicts are avoided.
+
+In some cases, especially when there are many packages already installed in a 
+conda environment, conda's built-in dependency solver can struggle to solve the 
+dependency graph, taking several to tens of minutes, and sometimes never 
+solving. In these cases, it is recommended to try `libmamba 
+<https://conda.github.io/conda-libmamba-solver/getting-started/>`_.
+
+To install and set the ``libmamba`` solver, run the following commands:
+
+.. prompt:: bash $, auto
+
+   \# Install miniconda
+   \# (can not use anaconda/miniconda not installed by you as installing libmamba 
+   \#  requires ownership over the anaconda/miniconda install directory)
+   $ wget https://repo.anaconda.com/miniconda/Miniconda3-py310_22.11.1-1-Linux-x86_64.sh
+   $ bash Miniconda3-py310_22.11.1-1-Linux-x86_64.sh
+
+   \# Install libmamba
+   $ conda install -n base conda-libmamba-solver
```

Comment:
```suggestion
   # Install miniconda
   # (you can not use the preinstalled anaconda/miniconda as installing libmamba
   #  requires ownership over the anaconda/miniconda install directory)
   $ wget https://repo.anaconda.com/miniconda/Miniconda3-py310_22.11.1-1-Linux-x86_64.sh
   $ bash Miniconda3-py310_22.11.1-1-Linux-x86_64.sh

   # Install libmamba
   $ conda install -n base conda-libmamba-solver
```

Reply:
Sorry about suggesting the removal of the `\#` to `#` only. This looks bad so I reverted